services/agent/tests/conftest.py

```diff
@@ -58,7 +59,7 @@ def node_uuid() -> str:
 
 @pytest.fixture
 def run_id() -> str:
-    return f"{uuid4()}"
+    return f"{arrow.utcnow().int_timestamp}"
```

Comment:
Q: In theory if we run director-v2 in the future in more replicas, is there some chance that two dynamic services will be created at the exact same time? I guess the probability is really low but still, I would maybe add also some randomness to the name. (I can imagine when e2e is creating a lot of parallel tests, in theory, we might get unlucky)

Reply:
This will not be a problem. You never handle a crete resource event in parallel.
You will always handle it in one single worker. From my point of view this concern will not apply.
Also you cannot start the same dynamic-sidecar if it is already running.

Reply:
The id will be for sure unique, since it's the linux timestamp to second accuracy.

Reply:
Yes, exactly "to the second accuracy" is my concern, as I can imagine two dynamic sidecars can be created during the same second. What I am trying to say I would be more careful here and not misuse the ID field, but rather create a new one with a timestamp if it is needed. 

https://softwareengineering.stackexchange.com/questions/305904/how-likely-are-collisions-of-timestamp-based-identifiers

Reply:
But as mentioned before, if you see benefits to having it there because of the more accurately track sorting of the data associated with each run it's fine. I would just then for the resource usage tracker purpose prefer more standard, ex. uuid format unique ID (and we can create a new field for that if necessary).

Reply:
Even if you were to have 2 requests to start in parallel the same dynamic service, one of them would fail. The code creating the run_id will never be ran.

So while it is a perfectly reasonable concern, it does not apply.

Reply:
Please also check the questions from PC and SAN

Reply:
Please note that the current version of the `RunID` is an extension of the previous one and no longer contains only the timestamp.

Reply:
Cool thanks, fine by me :+1: